components/org.wso2.carbon.appmgt.mdm.osgiconnector/src/main/java/org/wso2/carbon/appmgt/mdm/osgiconnector/mdmmgt/beans/ios/AppStoreApplication.java

```diff
@@ -0,0 +1,87 @@
+/*
```

Comment:
There are some common properties /mdmmgt/beans/ios/* classes. Better to introduce the super class.